src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicStream.cs

```diff
@@ -376,7 +378,7 @@ private CancellationTokenRegistration HandleWriteStartState(bool emptyBuffer, Ca
                         throw new QuicStreamAbortedException(_state.SendErrorCode);
                     }
 
-                    throw new OperationCanceledException(SR.net_quic_sending_aborted);
+                    throw new QuicOperationAbortedException();
```

Comment:
Any reason for not providing a message here?

Reply:
no reason, I forgot to put it there when copying